docs/user-guide/release-notes/2.3.x.rst

```diff
@@ -0,0 +1,12 @@
+2.3 Release Notes
+=================
+
+2.3.0
+-----
+
+See the :fixedbugs:`2.3.0`.
+
+A new command, docker repo tag, can point tags to manifests. It accepts
```

Comment:
The command should be decorated with double back ticks: ` ``docker repo tag`` `.